ff/serving/runner/kubernetes.go

```diff
@@ -0,0 +1,168 @@
+package runner
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/uuid"
+	batchv1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	watch "k8s.io/apimachinery/pkg/watch"
+	kubernetes "k8s.io/client-go/kubernetes"
+	rest "k8s.io/client-go/rest"
+)
+
+func generateKubernetesEnvVars(envVars map[string]string) []v1.EnvVar {
+	kubeEnvVars := make([]v1.EnvVar, len(envVars))
+	i := 0
+	for key, element := range envVars {
+		kubeEnvVars[i] = v1.EnvVar{Name: key, Value: element}
+		i++
```

Comment:
Unused

Reply:
Are you referring to the i variable? Should I use the append list convention instead?